README.md

```diff
@@ -1,8 +1,83 @@
-# hackbot
 ## Discord bot for Hackathon
-### This bot is a continuing project and I welcome pull requests that improve/fix or add features
+This bot is a continuing project and I welcome pull requests that improve/fix or add features
 
+
+
+
+
+
+
+## Installation Instructions
+
+Prerequisites:
+
+Linux / WSL
```

Comment:
Just so you're aware, this project doesn't rely on linux or any linux specific tools. *Almost* all python projects are OS agnostic. The only reason we had you set up a WSL instance is because it's  easier for us linux heads to set that up.

If you'd like to change this portion, you're welcome to it, but these changes are fantastic. I'll be merging it in soon either way.